### DISCUSSION TITLE
FEAT: Unwrap/merge for text based components

### DISCUSSION THREAD

**Discussion by bernesto** [GrapesJS/grapesjs#5725] [Roadmap Suggestions]
I am thinking that we need an unwrap function in the toolbar for inline block elements such as `span`, `b`, `i`, `em`, `link`, etc. 

Currently a user can't merge these elements into the outer content without deleting and recreating them. A good example is a user that wants to unlink a hyperlink. Or, a scenario where imported HTML with inline styles like `span`, `b`, `i`, `em`, or yuck `font`. 

It would even be better to have strip formatting button on parent containers too do this in mass.

Open to thoughts.

**Comment by artf** [comment]
Ok, let's move this one to the Roadmap Suggestions.
